openshift_images/cnf-building-and-deploying-a-dpdk-payload.adoc

```diff
@@ -69,17 +73,35 @@ fi
 while true; do sleep inf; done;
 ----
 
+The example `run.sh` will run the commands inside the `test-template.sh`.
+
+----
+spawn ./customtestpmd -l ${CPU} -w ${PCIDEVICE_OPENSHIFT_IO_DPDKNIC}
+--iova-mode=va -- -i --portmask=0x1 --nb-cores=2 --forward-mode=mac --port-topology=loop
+--no-mlockall
+set timeout 10000
+expect "testpmd>"
+send -- "start\r"
+sleep 20
+expect "testpmd>"
+send -- "stop\r"
+expect "testpmd>"
+send -- "quit\r"
+expect eof
+----
+
+This file will run the `testpmd` compiled application from the build stage.
+We spawn the `testpmd` interactive terminal then start a test workload and close it after 20 seconds.
```

Comment:
s/We spawn/Spawn  or This spawns ?
terminal, then

Reply:
"This spawns"